CHANGELOG.md

```diff
@@ -4,10 +4,12 @@
 
 ### Adds
 
+* Dynamic choice functions in schemas now recieve their original doc id for further inspection by your function.
```

Comment:
Make sure you request a docs update from Bob (or just PR one).

Reply:
yep he's got one